invenio_app_ils/records/jsonresolver/circulation.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2019 CERN.
+#
+# invenio-app-ils is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Ils Records Documents record circulation resolver."""
+
+import jsonresolver
+from invenio_circulation.api import get_loan_for_item, is_item_available
+from invenio_circulation.search.api import search_by_pid
+from werkzeug.routing import Rule
+
+from invenio_app_ils.circulation.utils import circulation_is_item_available, \
+    circulation_items_retriever
+from invenio_app_ils.records.api import Document
+
+
+@jsonresolver.hookimpl
+def jsonresolver_loader(url_map):
+    """Resolve the circulation for document record."""
+    from flask import current_app
+
+    def _circulation_for_document_resolver(document_pid):
+        """Return circulation info for the given document."""
+        circulation = {}
+        active_loans = 0
+        loanable_items = 0
+        number_of_items = 0
```

Comment:
Personal convention I follow, whenever I have to count something I postfix it with _count in order avoid numbers_of, num_of etc... in the particular example `items_count`.